packages/frontend-asset-loader/src/index.ts

```diff
@@ -0,0 +1,53 @@
+
+import fg from "fast-glob";
+import { Express } from "express";
+import {
+  getDuplicateHashedFileName,
+  isValidHashName,
+} from "./utils/utils";
+import { PathAndFile } from "./utils/utils.types";
+import { getLogger, CustomLogger, setLogger } from "./utils/logger";
+
+export const loadAssets = (
+  app: Express,
+  assetPath: string,
+  hashBetween = { start: "-", end: "." },
+  customLogger?: CustomLogger
+) => {
+  setLogger(customLogger);
+  const assets = fg.sync(assetPath);
+  const pathsAndFiles = parseAssets(assets, hashBetween);
+  const logger = getLogger();
+
+  const duplicateFileName = getDuplicateHashedFileName(pathsAndFiles);
+  if (duplicateFileName) {
+    logger.warn(
+      `Warning: Duplicate asset name detected. Each asset must have a unique name. Duplicate found: ${duplicateFileName}`
+    );
+  }
+
+  mapAssetsToLocal(app, pathsAndFiles);
+};
+
+export const parseAssets = (assets: string[], hashBetween: { start: string; end: string }): PathAndFile[] => {
```

Comment:
Sorry, small one but can we move these two functions (`parseAssets` & `mapAssetsToLocal`) in to another file so that they're not exposed to the teams using the package.